modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -39,7 +39,7 @@ Ensure that the host project applies one of the following configurations to the
 * `roles/compute.networkUser`
 ====
 
-If you do not supply a service account for control plane nodes in the `install-config.yaml` file, please grant the below permissions to the service account in the host project.
+If you do not supply a service account for control plane nodes in the `install-config.yaml` file, please grant the below permissions to the service account in the host project. If you do not supply a service account for control plane nodes or compute nodes in the `install-config.yaml` file, please grant the below permissions to the service account in the host project for cluster destruction.
```

Comment:
>  If you do not supply a service account for control plane nodes or compute nodes in the `install-config.yaml` file, please grant the below permissions to the service account in the host project for cluster destruction.

As the last sentence has mentioned `a service account for control plane nodes`, shall this sentence mention only `a service account for compute nodes...for cluster destruction`, to avoid repetition? Thanks!